spec.html

```diff
@@ -33139,7 +33139,7 @@
       <!-- es6num="24.2.1.1" -->
       <emu-clause id="sec-getviewvalue" aoid="GetViewValue">
         <h1>GetViewValue ( _view_, _requestIndex_, _isLittleEndian_, _type_ )</h1>
-        <p>The abstract operation GetViewValue with arguments _view_, _requestIndex_, _isLittleEndian_, and _type_ is used by functions on DataView instances is to retrieve values from the view's buffer. It performs the following steps:</p>
+        <p>The abstract operation GetViewValue with arguments _view_, _requestIndex_, _isLittleEndian_, and _type_ is used by functions on DataView instances to retrieve values from the view's buffer. It performs the following steps:</p>
```

Comment:
"The operation is used by functions is to retrieve values" sounds ungrammatical to me, but "ungrammatical" was too long for the first line of the commit message.